Yosemite/Yosemite/Actions/BlazeAction.swift

```diff
@@ -5,6 +5,18 @@ import Networking
 //
 public enum BlazeAction: Action {
 
+    /// Creates a new Blaze campaign
+    ///
+    /// - `campaign`: New Blaze campaign to be created
+    /// - `siteID`: the site in which Blaze campaign should be created.
+    /// - `onCompletion`: invoked when the create operation finishes.
+    ///     - `result.success(String)`: ID of the newly created Blaze campaign.
+    ///     - `result.failure(Error)`: error indicates issues creating a Blaze campaign.
+    ///
+    case createCampaign(campaign: CreateBlazeCampaign,
+                        siteID: Int64,
+                        onCompletion: (Result<String, Error>) -> Void)
```

Comment:
I don't think we'll use the ID on the UI. Should we return Void for the success case for simplicity?

Reply:
Update in https://github.com/woocommerce/woocommerce-ios/pull/11620/commits/7dabc23407e2d0dd08a67f5551fbd8fd17684787